client/tests/integration/domain_owner_permissions.rs

```diff
@@ -325,7 +325,7 @@ fn domain_owner_trigger_permissions() -> Result<()> {
     let bob_id: AccountId = "bob@wonderland".parse()?;
     let token = PermissionToken::new(
         "CanUnregisterUserTrigger".parse().unwrap(),
-        &json!({ "trigger_id": trigger_id }),
+        &json!({ "account_id": bob_id }),
```

Comment:
now this gives a permission to remove any trigger, instead of only one

Reply:
Yeah, we discussed this with @Stukalov-A-M and decided that it make sense to feel `CanRegister/CanUnregister` permissions more uniformly. 

Like there is not much value in token `CanRegisterUserTrigger` which accepts trigger id.